gtsam/hybrid/HybridGaussianFactorGraph.h

```diff
@@ -85,7 +87,7 @@ struct EliminationTraits<HybridGaussianFactorGraph> {
   /// The default ordering generation function
   static Ordering DefaultOrderingFunc(
       const FactorGraphType& graph,
-      boost::optional<const VariableIndex&> variableIndex) {
+      const VariableIndex&) {
```

Comment:
Same comment: should be std::reference_wrapper. No idea why HybridOrdering would not use it if available.